src/functions/tester/mod.ts

```diff
@@ -64,3 +78,42 @@ export const SlackFunctionTester: SlackFunctionTesterFn = <
 
   return { createContext };
 };
+
+export const SlackActionHandlerTester: SlackActionHandlerTesterFn = <
```

Comment:
I find it a bit odd that this internal only test helper lives here in `functions/tester` which is intended for devs to test their functions. I'd like to either move this somewhere else specifically for our tests, or add a comment calling out this is internal only and therefore dissimilar to `SlackFunctionTester` above

Reply:
For now, I'll move it into the action router tests, since that's the only place it is used, and we can revisit if/how we want to expose it to devs down the road.